index.bs

```diff
@@ -4767,13 +4766,11 @@ listed in the [=error names table=] below.
 The table also indicates the {{DOMException}}'s integer code for that error name,
 if it has one.
 
-There are two types that can be used to refer to exception objects:
-{{DOMException}} and {{Error!!interface}},
-where the latter encompasses [=simple exceptions=] and {{DOMException}}.
+There is one type that can be used to refer to exception objects:
+{{DOMException}}.
```

Comment:
I don't think this is a type any more? Or maybe it's just an interface type. In which case I think this paragraph is confusing; everything defined as an interface is an interface type. A paragraph like this makes me think I should find special conversion rules for the type.